tests/integration.rs

```diff
@@ -2611,16 +2624,21 @@ mod common_parallel {
         let mut cmd = GuestCommand::new(&guest);
         cmd.args(["--cpus", "boot=2,max=4"])
             .args(["--memory", "size=512M"])
-            .args(["--kernel", direct_kernel_boot_path().to_str().unwrap()])
-            .args(["--cmdline", DIRECT_KERNEL_BOOT_CMDLINE])
             .capture_output()
             .default_disks()
             .default_net();
 
+        let kernel = direct_kernel_boot_path();
```

Comment:
I think you should just add a new test rather than trying to repurpose this one.

Reply:
I am going to add more test , that is 10-20 test cases and I wanted to rather reuse the test cases than dupicating the code.Only difference here is to use IGVM file instead of bzImage and 2 other params that I can conditionally add.